src/DetailsView/components/adhoc-issues-test-view.tsx

```diff
@@ -2,15 +2,15 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { NamedSFC } from '../../common/react/named-sfc';
+import { NamedFC } from '../../common/react/named-sfc';
```

Comment:
Should probably rename the file named-sfc.ts -> named-fc.ts to match

Reply:
updated.